### PR TITLE
🎨 Format placeholder for trade input fields

### LINKED ISSUE #876: Fix: Format Trade input form placeholder
# ℹ Overview
Format trade input placeholder

### PR DESCRIPTION
**Pull request by IYTEC**
# ℹ Overview
Format placeholder in trade input fields

### 📝 Related Issues
- closes #876 
- 

### 🔐 Acceptance:

- [ ] `yarn test` passes

### CODE REVIEW

File: components/Wallet/PlaceOrder/Form/TradeInputs.jsx [hunk 1]
```diff
@@ -155,7 +155,7 @@ export const TradeInputs = ({
         }}
         value={order.price || ''}
         onChange={handleChange}
-        placeholder='0.00'
+        placeholder={`0.${'0'.repeat(Math.max(0, 6 - 4))}`}
```
Comment (atrefonas) [inline components/Wallet/PlaceOrder/Form/TradeInputs.jsx]: This one can just be hard-coded to 0.00 Since it is always the same

**Review by atrefonas — CHANGES_REQUESTED** [review]

**Review by atrefonas — CHANGES_REQUESTED** [review]
See comment

File: components/Wallet/PlaceOrder/Form/TradeInputs.jsx (hunk 1 above)
Comment (IYTEC) [reply components/Wallet/PlaceOrder/Form/TradeInputs.jsx]: Yes, we can change it back when we do inversion.

**Review by atrefonas — CHANGES_REQUESTED** [review]
When it is 0 decimals, the placeholder should be "0" not "0."

**Review by atrefonas — APPROVED** [review]

**Pull request merged**